tests/cells_by_location_marker.yaml

```diff
@@ -0,0 +1,47 @@
+id: cells_by_location_marker
+description: This tests all cells in location X expressing marker Z
+
+input:
+  locations:
+    - http://purl.obolibrary.org/obo/UBERON_0002113
+  markers:
+    - http://identifiers.org/hgnc/4172
+
+expected_result:
+  cell:
+    - http://purl.obolibrary.org/obo/CL_1000716
+    - http://purl.obolibrary.org/obo/CL_1000454
+    - http://purl.obolibrary.org/obo/CL_1000768
+
+query_mechanism:
+  relationgraph:
+    query: "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+      PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+      PREFIX owl: <http://www.w3.org/2002/07/owl#>
+      PREFIX expresses: <http://purl.obolibrary.org/obo/RO_0015004>
+      PREFIX part_of: <http://purl.obolibrary.org/obo/BFO_0000050>
+      PREFIX cell: <http://purl.obolibrary.org/obo/CL_0000000>
+      PREFIX has_characterizing_marker_set: <http://purl.obolibrary.org/obo/RO_0015004>
```

Comment:
why do we need this + expresses when we can infer expresses from the property chain we just added?

Reply:
Ops forgot to remove it.

Reply:
THought you might have as see 'expresses' used elsewhere. 

BTW - what do the reports look like or do these still need work?  Couldn't spot anything in GA readout.   Might be best for users if we could save a reporting log to repo.

Reply:
The output is like this:

<img width="599" alt="Screenshot 2023-01-30 at 13 09 45" src="https://user-images.githubusercontent.com/2208124/215486734-24e6c6cb-265e-461c-b345-b1e00fff29c6.png">

I can put that in a file.